lib/active_merchant/billing/gateways/fortis.rb

```diff
@@ -110,7 +110,7 @@ class FortisGateway < Gateway
       }
 
       def initialize(options = {})
-        requires!(options, :user_id, :user_api_key, :developer_id)
+        requires!(options, :user_id, :user_api_key)
```

Comment:
Since `developer_id` is no longer a required field, should we also remove it from the fixtures? ❓

Reply:
It's still "required" just not for the merchant. For now, I think we can leave it because it's loaded on our side via an env var.